cabinets/__init__.py

```diff
@@ -41,7 +42,9 @@ def from_uri(uri) -> (Cabinet, str):
     try:
         protocol, path = uri.split('://')
     except ValueError:
-        raise InvalidURIError("Missing protocol identifier")
+        debug("No protocol identifier specified: using 'file'")
+        protocol = 'file'
+        path = uri
```

Comment:
For parity with the `try` block
```suggestion
        protocol, path = 'file', uri
```